src/main/java/com/templateproject/api/controller/ArticleController.java

```diff
@@ -0,0 +1,65 @@
+package com.templateproject.api.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.templateproject.api.OffsetBasedPageRequest;
+import com.templateproject.api.entity.Article;
+import com.templateproject.api.repository.ArticleRepository;
+
+@RestController
+public class ArticleController {
+
+    @Autowired
+    ArticleRepository articleRepository;
+
+    @GetMapping("/articles")
+    public List<Article> getAllEmployees(int limit, int offset) {
+        Pageable pageable = new OffsetBasedPageRequest(limit, offset);
+        return articleRepository.findAll(pageable).getContent();
+    }
+
+    @GetMapping("/articles/{id}")
+    public Article show(@PathVariable int id) {
+        return articleRepository.findById(id).get();
+    }
+
+    @PostMapping("/articles")
+    public Article create(@RequestBody Article article) {
+        return articleRepository.save(article);
```

Comment:
When you create an article you need to set the two date fields in the constructor. It won't be send from the front